app/mailers/supervisor_mailer.rb

```diff
@@ -19,4 +19,13 @@ def weekly_digest(supervisor)
       subject: "Weekly summary of volunteers' activities for the week of #{Date.today - 7.days}"
     )
   end
+
+  def reimbursement_request_email(volunteer, supervisor)
+    @volunteer = volunteer
+    @casa_organization = volunteer.casa_org
+    @supervisor = supervisor
+    if supervisor.receive_reimbursement_email
+      mail(to: @supervisor.email, subject: "New reimbursement request from #{@volunteer.display_name}")
+    end
```

Comment:
Good use of an if conditional to make sure that receieve_reimbursement_email is part of the supervisor parameter before sending mail to the supervisor email